TypeCobol/Tools/Options-Config/TypeCobolConfiguration.cs

```diff
@@ -232,64 +204,25 @@ public static OptionSet GetCommonTypeCobolOptions(TypeCobolConfiguration typeCob
 
             //// Options values verification
             //InputFileError
-            foreach (var path in config.InputFiles)
-            {
-                if (FileSystem.GetFiles(path, recursive: false).IsNullOrEmpty() && !errorStack.ContainsKey(ReturnCode.InputFileError))
-                {
-                    errorStack.Add(ReturnCode.InputFileError, TypeCobolConfiguration.ErrorMessages[ReturnCode.InputFileError]);
-                }
-            }
+            VerifFiles(config.InputFiles, ReturnCode.InputFileError, ref errorStack);
+            
             //SkeletonFileError
             if (config.ExecToStep == ExecutionStep.Generate && !config.skeletonPath.IsNullOrEmpty() &&  !errorStack.ContainsKey(ReturnCode.ExecToStepError))
             {
-                if (!File.Exists(config.skeletonPath))
-                {
-                    errorStack.Add(ReturnCode.SkeletonFileError, TypeCobolConfiguration.ErrorMessages[ReturnCode.SkeletonFileError]);
-                }
+                VerifFiles(new List<string> { config.skeletonPath }, ReturnCode.SkeletonFileError, ref errorStack);
```

Comment:
2 options here : 
- Create a method for a signle file that could be use by VerifFiles()
- Do not use VerifFiles() and simply do the file check here